src/components/Sum.jsx

```diff
@@ -1 +1,14 @@
 // export function Sum() {}
+import React from 'react';
+
+export const Sum = ({
+  a,
+  b,
+}) => (
+  <p>{`Sum of ${a} and ${b} is ${a + b}`}</p>
```

Comment:
It's better to move the text to the new line, it is more readable this way
```suggestion
  <p>
    {`Sum of ${a} and ${b} is ${a + b}`}
  </p>
```